### ISSUE TITLE
[BUG]: Intermitent audio

### ISSUE THREAD

**Issue by gaspo53** [odioapp/odio#51]
**Describe the bug**
Randomly, the audio stops, and then you need to press play/pause twice
**Steps to reproduce the behavior**
Just leave it running
**Expected behavior**

**Desktop (please complete the following information):**
 - OS: Windows 10/macOS Mojave
 - Version [e.g. Mojave]

**Comment by probonopd** [comment]
Which exact radio station?

**Comment by spiral2k** [comment]
@gaspo53 still happening?

**Comment by alexwender1** [comment]
I also noticed this problem. I did not detect a pattern ... only the radio stops working; then reopened the application and ran normally.

I only detected this anomaly in the latest version ... to 1.4.0.

**Comment by spiral2k** [comment]
@alexwender1 @gaspo53  when the audio stops, what the state of the the play/pause button?

**Comment by gaspo53** [comment]
@spiral2k @alexwender1 yes, the audio simply stops and the UI it's "playing", you need to click pause, then play again, and sometimes you need to click in the radio again (in the search results pane), otherwise it won't play anything.

**Comment by spiral2k** [comment]
The problem was that when buffering after load (slow internet connection or slow download the stream data) i have not updating the buffering animation :(.

in the next version odio will show the correct animation when buffering.

**Comment by Xplouder** [comment]
This is happening to me really often. Maybe a solution would be to add a delay time to prevent this, this way we could play the stream in the buffer until it restores the connection/fills up the buffer again.

**Comment by spiral2k** [comment]
Fixed that already, wait to the next version :)

**Comment by Xplouder** [comment]
Great! Any release ETA? :)
